tests/testthat/test-submodule.R

```diff
@@ -93,43 +93,47 @@ test_that("Can install a repo with a submodule", {
       "-c", "user.name=foobar", "-c", paste0("user.email=", shQuote("<>")),
       "commit", "-m", shQuote("Initial commit")))
   })
-  module <- file.path("submodule", ".gitmodules")
-  on.exit(unlink(module), add = TRUE)
+
+  sub <- tempfile()
+  dir.create(sub)
+  on.exit(unlink(sub,recursive=TRUE,force=TRUE),add=TRUE)
+
+  module <- file.path(sub, ".gitmodules")
```

Comment:
Are these changes necessary?

Reply:
The test wasn't passing locally without these changes. `writeLines()` couldn't create the directory, so I used tempfile() to do so.